graph.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+# graph.py
+from collections import defaultdict
+
+
+class Graph():
+    def __init__(self, vertices):
+        self.graph = defaultdict(list)
+        self.V = vertices
+
+    def add_edge(self, u, v):
+        self.graph[u].append(v)
+    #internal
+    def is_cyclic_util(self, v, visits_p,n):
```

Comment:
There are a lot of stylistic issues like putting a space after the comma, around `=`, etc. Instead of making comments for all of them I would recommend that your code passes [a flake8 check](http://flake8.pycqa.org/en/latest/).